Examples/Scripts/Python/truth_tracking_kalman.py

```diff
@@ -73,7 +73,6 @@ def runTruthTrackingKalman(
                 outputParticles="particles_input",
             )
         )
-        s.addWhiteboardAlias("particles", "particles_input")
```

Comment:
@andiwand Do you remember if that alias is actually needed? We have it only in the KF but do not even use it here. Potentially, we should also add it to the other chains then.

Reply:
I think we were using `"particles"` in some downstream truth matching or simulation at some point. As far as I can see this should be `"particles_selected"` these days and `"particles"` might not be used anymore.

I think this is one of the paths in the script we never test?